content/en/docs/contribute/localization.md

```diff
@@ -41,13 +44,54 @@ git clone https://github.com/<username>/website
 cd website
 ```
 
-### Open a pull request
+The localization you want to help out with is inside `content/<two-letter-code>`.
```

Comment:
Line 47, could you add more of an intro. Something like:
The website content directory includes sub-directories for each language.
...

Reply:
Done, I think.